package.json

```diff
@@ -60,18 +59,13 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.2.5",
-    "@babel/runtime": "^7.2.0",
```

Comment:
Removing these probably wouldn't create a build error, but are we sure they weren't modifying the code during build to help hit a build target or pollyfill behavior not available in Safari/IE11?

Reply:
For example, `@babel/polyfill` is required for `redux-oidc` to work:
https://github.com/maxmantz/redux-oidc#babel-polyfill

Reply:
Good to know. Thank you for sharing that @dannyrb